reana-ui/src/pages/workflowList/components/WorkflowList.js

```diff
@@ -0,0 +1,209 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2020 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import React, { useEffect, useState, useRef } from "react";
+import axios from "axios";
+import moment from "moment";
+import { Dimmer, Icon, Loader, Container, Popup } from "semantic-ui-react";
+
+import NoWorkflows from "./NoWorkflows";
+import config from "../../../config";
+
+import styles from "./WorkflowList.module.scss";
+import Title from "../../../components/Title";
+
+export default function WorkflowList() {
+  const [workflows, setWorkflows] = useState(null);
+  const [loading, setLoading] = useState(false);
+  const interval = useRef(null);
+
+  useEffect(() => {
+    /**
+     * Gets data from the specified API
+     */
+    function getWorkflows() {
+      setLoading(!interval.current);
+      axios({
+        method: "get",
+        url: config.api + "/api/workflows",
+        withCredentials: true
+      }).then(res => {
+        setWorkflows(parseData(res.data));
+        setLoading(false);
+      });
+    }
+
+    /**
+     * Parses API data into displayable data
+     */
+    function parseData(workflows) {
+      if (!Array.isArray(workflows)) return [];
+      workflows.sort((a, b) => a.created < b.created);
```

Comment:
I am testing this out and the order seems to not be correct. What about [comparing with `moment.js`](https://momentjs.com/docs/#/query/) after the `forEach`?